src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
See https://bugs.chromium.org/p/chromium/issues/detail?id=706561 which removes `hasOrientation` among other things. For `VRDisplay`, but will do likely do same for Gamepad Extensions so do need to rely on that flag? At least `hasPosition` false case might be better